ui/src/hooks/vehicle-service/useGetTimetables.ts

```diff
@@ -208,6 +208,10 @@ export const useGetTimetables = (journeyPatternId: UUID) => {
   const [timetables, setTimetables] = useState<TimetableWithMetadata>();
 
   const getTimetablesForRoute = useCallback(async () => {
+    if (!journeyPatternId) {
+      return;
+    }
+
     const res = await getTimetablesForOperationDay({
```

Comment:
this is a bad name for this function... is this rather getRouteTimetablesForOperationDay?